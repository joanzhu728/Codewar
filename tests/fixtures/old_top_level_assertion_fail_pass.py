import codewars_test as test

test.assert_equals(1, 2)
test.assert_equals(1, 1)
