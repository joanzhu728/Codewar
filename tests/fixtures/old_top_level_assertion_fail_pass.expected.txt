
<FAILED::>1 should equal 2

<PASSED::>Test Passed
